### PR TITLE
Feat: Create TILT Spec and endpoint

### LINKED ISSUE #54: Create TILT Spec

### PR DESCRIPTION
**Pull request by akkbng**
Fixes #54 and #55 

- Removed the overall `/user` prefix and add to the routes individually, because of the admin routes
- /user_service/tilt/user_service_tilt.py --> python file to generate `dataDisclosed` TILT spec. Values in there are arbitrary (took them from the example spec in TILT repo), currently only "Language Preference" of the user is there as disclosed data. We don't have to use this type of generator and I don't know if it's useful.
- /user_service/tilt/user_service_tilt.json --> generated json file
- [GET endpoint for Tilt Spec](https://github.com/mindtastic/user-service/blob/78f3befdaae95f9791d14c28a1941fcc0cc70598/user_service/routes/user_routes.py#L140) Currently this route reads the generated json file and returns that, but this of course can be improved.

### CODE REVIEW

**Comment by jgraeger** [comment]
Suggested edit:
```diff
diff --git a/user_service/tilt/user_service_tilt.json b/user_service/tilt/user_service_tilt.json
deleted file mode 100644
index 3a01044..0000000
--- a/user_service/tilt/user_service_tilt.json
+++ /dev/null
@@ -1,61 +0,0 @@
-{
-    "dataDisclosed": {
-        "_id": "user-service-tilt-01",
-        "category": "Language Preference",
-        "legalBases": [
-            {
-                "description": "General Data Protection Regulation (GDPR) Art.",
-                "reference": "GDPR-99-1-a"
-            },
-            {
-                "description": "BDSG-42-5 refers to the processing of personal data within..",
-                "reference": "BDSG-42-5"
-            }
-        ],
-        "legitimateInterests": [
-            {
-                "exists": false,
-                "reasoning": "No legitimate interests"
-            }
-        ],
-        "nonDisclosure": {
-            "consequences": "If the data is not disclosed, the shipment cannot be delivered.",
-            "contractualRegulation": false,
-            "legalRequirement": false,
-            "obligationToProvide": false
-        },
-        "purposes": [
-            {
-                "description": "To provide the user-service-tilt",
-                "purpose": "To provide the user-service-tilt"
-            }
-        ],
-        "recipients": [
-            {
-                "address": "",
-                "category": "",
-                "country": "DE",
-                "division": "",
-                "name": "",
-                "representative": null
-            }
-        ],
-        "storage": [
-            {
-                "aggregationFunction": "min",
-                "legalBasisConditional": [
-                    "SGB-100-42"
-                ],
-                "purposeConditional": [
-                    "Data is always stored"
-                ],
-                "temporal": [
-                    {
-                        "description": "Temporal element for the user-service-tilt",
-                        "ttl": ""
-                    }
-                ]
-            }
-        ]
-    }
-}
\ No newline at end of file

```

**Comment by jgraeger** [comment]
Suggested edit:
```diff
diff --git a/user_service/routes/user_routes.py b/user_service/routes/user_routes.py
index c6a7a17..3ca094d 100644
--- a/user_service/routes/user_routes.py
+++ b/user_service/routes/user_routes.py
@@ -1,4 +1,3 @@
-import json
 from typing import List, Union
 from uuid import UUID
 from user_service.routes.dependencies import get_mongo_collection, ServiceDBCollection
diff --git a/user_service/tilt/user_service_tilt.py b/user_service/tilt/user_service_tilt.py
index 646590a..2ad9084 100644
--- a/user_service/tilt/user_service_tilt.py
+++ b/user_service/tilt/user_service_tilt.py
@@ -1,7 +1,4 @@
 from tilt import tilt
-import json
-from hashlib import sha256
-from datetime import datetime
 
 #List of legal_bases with reference and description
 first_legal_base = tilt.AnyOfSchemaForTheLegalBasesOfTheDataDisclosed(
@@ -64,7 +61,4 @@ disclosed_data = tilt.DataDisclosedElement(
 )
 
 tilt_dict = {}
-tilt_dict['dataDisclosed'] = disclosed_data.to_dict()
-
-# with open('user_service_tilt.json', 'w') as fp:
-#     json.dump(tilt_dict, fp, indent=4)
+tilt_dict['dataDisclosed'] = [disclosed_data.to_dict()]

```

File: user_service/routes/user_routes.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import json
```
Comment (jgraeger) [inline user_service/routes/user_routes.py:1]:
```suggestion
```
Unused import

File: user_service/tilt/user_service_tilt.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+from tilt import tilt
+import json
+from hashlib import sha256
+from datetime import datetime
```
Comment (jgraeger) [inline user_service/tilt/user_service_tilt.py:4]:
```suggestion
```
Unused imports.

File: user_service/tilt/user_service_tilt.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    legal_bases=[first_legal_base, second_legal_base],
+    storage=[storage],
+    legitimate_interests=[legitimate_interests],
+    non_disclosure=non_disclosure,
+    purposes=[purposes],
+    recipients=[first_recipient]
+)
+
+tilt_dict = {}
+tilt_dict['dataDisclosed'] = disclosed_data.to_dict()
```
Comment (jgraeger) [inline user_service/tilt/user_service_tilt.py:67]:
```suggestion
tilt_dict['dataDisclosed'] = [disclosed_data.to_dict()]
```
dataDisclosed is supposed to be an array.

Comment (jgraeger) [inline user_service/tilt/user_service_tilt.py:67]: I would add validation of the created scheme, like it is in the [example](https://github.com/Transparency-Information-Language/python-tilt#validate-documents) of the python-tilt repo. We are in an academic context, so implementation is left as an exercise to the reader

**Review by jgraeger — APPROVED** [review]
Take a look at the comments and suggestions, but from my side you're good to go 🚛

File: user_service/tilt/user_service_tilt.py (hunk 3 above)
Comment (akkbng) [reply user_service/tilt/user_service_tilt.py:67]: Yeap of course sorry, I missed it!

Comment (akkbng) [reply user_service/tilt/user_service_tilt.py]: Yeap, adding one now for the 'dataDisclosed' section from the full TILT schema 👍  Thanks!

**Pull request merged**